Description: Fixes the compatibility with the version of Plexus Containers in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/src/test/java/org/codehaus/plexus/archiver/DuplicateFilesTest.java
+++ b/src/test/java/org/codehaus/plexus/archiver/DuplicateFilesTest.java
@@ -31,7 +31,7 @@
         throws Exception
     {
         super.setUp();
-        getContainer().getLoggerManager().setThreshold( Logger.LEVEL_DEBUG );
+        ((org.codehaus.plexus.DefaultPlexusContainer) getContainer()).getLoggerManager().setThreshold( Logger.LEVEL_DEBUG );
     }
 
     public void testZipArchiver()
